config.js

```diff
@@ -6,8 +6,8 @@ function isTruthy(value) {
   return [true, 'true', 1, '1', 'yes', 'Y', 'on'].includes(value);
 }
 
-const ENABLE_DEBUG_FILE_WRITING = isTruthy(process.env.ENABLE_DEBUG_FILE_WRITING) || true;
-const ENABLE_SENDING_TO_VP_API = isTruthy(process.env.ENABLE_SENDING_TO_VP_API) || false;
+const ENABLE_DEBUG_FILE_WRITING = isTruthy(process.env.ENABLE_DEBUG_FILE_WRITING);
+const ENABLE_SENDING_TO_VP_API = isTruthy(process.env.ENABLE_SENDING_TO_VP_API);
 
```

Comment:
Let's take opportunity here to remove the test `VP_API_CLIENT_ID` and `SECRET` to an environment variable. 
We're getting close to deploy now, and will be testing with real data soon, and I'd rather have it out of this repo sooner than later.

As soon as we've done that we can ask Steve to invalidate these credentials and generate new ones for us, that won't be shared

Reply:
:+1: For ACC the credentials have already been changed (unrelated to this, VP moved their ACC instance to the a different host and the credentials had to change).

Will move this to an env var here.